src/features/ballot/components/BallotAllocationForm/ImportXLSX.tsx

```diff
@@ -13,6 +13,10 @@ export function ImportXLSX() {
   const save = useSaveBallot();
   const csvInputRef = useRef<HTMLInputElement>(null);
 
+  useEffect(() => {
+    // 
+  }, [votes, setVotes, save]);
+
```

Comment:
is this intentional?
what are you using it for?

Reply:
While testing this made it more responsive